src/main/resources/public/js/app.js

```diff
@@ -43,7 +95,10 @@ function connect() {
     stompClient = Stomp.over(socket);
     stompClient.connect({}, function (frame) {
         stompClient.subscribe(buildPath('notifications/process-instance'), function (message) {
-            handleMessage(JSON.parse(message.body));
+            handleProcessInstanceNotification(JSON.parse(message.body));
```

Comment:
It displays the info message "Process instance has changed." multiple times now. As a user, it's a bit annoying because I need the information only once :sweat_smile:   

![image](https://user-images.githubusercontent.com/4305769/149787439-c6671437-7bc0-4bae-a745-bf34762b2a4b.png)

Can we display the message only once?

Reply:
🤔 the reason for multiple occurrences is that they really happen ;)
Nevertheless, it's a bit annoying, true.
The simplest approach might be to check the previous message for a duplicate.
That should fix almost all of such use cases, would you agree? @saig0

Reply:
done. see latest commit.